pkg/plugins/platform/local.go

```diff
@@ -898,7 +895,7 @@ func (l *LocalApp) AddHostsEntry() error {
 	util.CheckErr(err)
 
 	fmt.Println("ddev needs to add an entry to your hostfile.\nIt will require root privileges via the sudo command, so you may be required\nto enter your password for sudo. ddev is about to issue the command:")
-	hostnameArgs := []string{ddevFullpath, "hostname", l.HostName(), dockerIP}
+	hostnameArgs := []string{ddevFullpath, "hostname", dockerIP, l.HostName()}
```

Comment:
I understand that this corresponds to the order in /etc/hosts but I think it was easier to use the other way.